tools/amgh/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG ARCH="amd64"
+ARG OS="linux"
+FROM quay.io/prometheus/busybox-${OS}-${ARCH}:glibc
```

Comment:
Why do you need to the `glibc` variant?

Reply:
it was building a dynamic build, as I was using the go command directly initially without the `netgo` tag. I added it in `.promu.yml` . changed it to `latest` now.